backend/src/db/migrations/20251028120000_add-certificate-sync-table.ts

```diff
@@ -0,0 +1,35 @@
+import { Knex } from "knex";
+
```

Comment:
IGNORE PLACEMENT OF COMMENT

Minor but I think we should add a top margin for this one?
<img width="1507" height="785" alt="image" src="https://github.com/user-attachments/assets/ee32c0ad-bdf3-4b8a-aa26-b6cdb6e05284" />

Maybe we can also have a hyperlink here in the "PKI Syncs" to redirect them to that page upon clicking?